### PR TITLE
[WIP] Getting started with M3 and Terraform guide

### PR DESCRIPTION
**Pull request by gibbscullen**
**What this PR does / why we need it**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
@gibbscullen First point… Make sure you rebase from master before creating a new branch, it looks like you have pulled in unrelated changes into this PR including changes to the theme (which changed a while back), and there's also the hosting change you made included in this PR too.

File: site/content/quickstart/terraform.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+linkTitle: "Terraform"
```
Comment (ChrisChinchilla) [inline site/content/quickstart/terraform.md:2]: @gibbscullen I'm not sure if the quickstart section is the best section, maybe _integrations_?

File: site/content/quickstart/terraform.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Install [Terraform CLI](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+
+**Note:** Recommended to use Terraform version 0.13.x 
+**Note:** Recommended to use M3 version 1.0 or higher
+
+## Step 1 - Bootstrap Terrafrom and M3 
+
+```bash
+$ mkdir terraform-project && cd terraform-project
```
Comment (ChrisChinchilla) [inline site/content/quickstart/terraform.md:17]:
```suggestion
mkdir terraform-project && cd terraform-project
```

File: site/content/quickstart/terraform.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+``` 
+
+
+## Step 2 - Configure M3 and initialize Terraform 
+
+
+```bash
+$ terraform init
```
Comment (ChrisChinchilla) [inline site/content/quickstart/terraform.md:25]:
```suggestion
terraform init
```

File: site/content/quickstart/terraform.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+## Pre-requisites: 
+
+Install [Terraform CLI](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+
+**Note:** Recommended to use Terraform version 0.13.x 
+**Note:** Recommended to use M3 version 1.0 or higher
+
+## Step 1 - Bootstrap Terrafrom and M3 
+
+```bash
```
Comment (ChrisChinchilla) [inline site/content/quickstart/terraform.md:16]:
```suggestion
```shell
```

And this is one I will add to the nearly finished style guide actually…

File: site/content/quickstart/terraform.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+$ mkdir terraform-project && cd terraform-project
+``` 
+
+
+## Step 2 - Configure M3 and initialize Terraform 
+
+
+```bash
```
Comment (ChrisChinchilla) [inline site/content/quickstart/terraform.md:24]:
```suggestion
```shell
```

**Comment by ChrisChinchilla** [comment]
@gibbscullen Possibly also make this a draft PR too? Though I am not sure if you can once you've already marked it as non-draft 🤔

**Comment by gibbscullen** [comment]
Closing and will open new open in new file path.

**Pull request closed without merging**